admin-client/src/components/SiteMetadata.svelte

```diff
@@ -55,8 +55,16 @@
         </LabeledItem>
       {/if}
       <div>
-        <ExternalLink href={viewLink}>Live</ExternalLink>
-        <ExternalLink href={demoViewLink} className="padding-x-1">Demo</ExternalLink>
+        {#each siteLinks as siteLink, index}
+          <ExternalLink
+            href={siteLink.url}
+            className={index > 0 ? 'padding-left-1' : ''}
+          >
+            {siteLink.context}
+          </ExternalLink>
+        {/each}
+      </div>
+      <div>
```

Comment:
The extra `div` here puts the repo link on a new line which ends up a little squished. Probably fine to have the new line if we add a little padding.

Reply:
Also I can't see why but the underline continues past the word on each link? Not essential to fix unless you see something obvious

Reply:
I'll add some padding to the repo block to add space. I also wasn't able to quickly identify the underscore overhang. I'll take another look with the repo padding fixes.